ui/desktop/.ember-cli

```diff
@@ -11,5 +11,6 @@
   Setting `isTypeScriptProject` to true will force the blueprint generators to generate TypeScript
   rather than JavaScript by default, when a TypeScript version of a given blueprint is available.
   */
-  "isTypeScriptProject": false
+  "isTypeScriptProject": false,
+  "port": 4201
```

Comment:
None blocking comment: I didn't know we could delcare it here 😄 . I checked the `ui/admin/.ember-cli` and seems we do not have decalred there. Does admin uses `4200` without declaration because is the default port ember will use?

Reply:
Yes, 4200 is the default port for ember apps.